tests/arrow_tests.py

```diff
@@ -1244,6 +1244,15 @@ def test_days(self):
         assertEqual(self.now.humanize(later, only_distance=True), '2 days')
         assertEqual(later.humanize(self.now, only_distance=True), '2 days')
 
+        later = self.now.shift(days=3)
+        assertEqual(later.humanize(), 'in 3 days')
+
```

Comment:
Maybe add a comment why these tests are needed (reference this PR/issue), that way people in the future won't remove them as unnecessary.

Reply:
Hi @systemcatch, thanks for the input, I just added a comment in the most recent commit.